src/QsCompiler/LanguageServer/Program.cs

```diff
@@ -121,7 +126,6 @@ private static int Run(Options options)
             catch (Exception ex)
             {
                 Log("[ERROR] MsBuildLocator could not register defaults.", options.LogFile);
```

Comment:
```suggestion
                Log("[ERROR] MsBuildLocator could not register defaults.", options.LogFile);
                // Don't exit here, since exiting without establishing a connection will result in a cryptic failure of the extension. 
                // Instead, we proceed to create a server instance and establish the connection. 
                // Any errors can then be properly processed via the standard server-client communication as needed. 

```